src/openrct2-ui/windows/Options.cpp

```diff
@@ -908,11 +897,7 @@ namespace OpenRCT2::Ui::Windows
                 disabled_widgets &= ~(1uLL << WIDX_RESOLUTION_LABEL);
             }
 
-            // Disable Steam Overlay checkbox when using software or OpenGL rendering.
-            // TODO: Check if this even works properly.
-            disabled_widgets |= (1uLL << WIDX_STEAM_OVERLAY_PAUSE);
-
-            // VSync.
+            disabled_widgets &= (1uLL << WIDX_STEAM_OVERLAY_PAUSE);
```

Comment:
I think this should be inverted as well.

Reply:
Not sure I follow, do we want this option always on or always off? I don't even know if this works correctly.

Reply:
This option was only ever _unavailable_ for the non-hardware backed renderers, so I think it should just always be available now. (Not necessarily on, that's up to the player.)

I suppose we could query SDL to see if the canvas is actually hardware backed, but I personally think that's overkill.

Reply:
So what are you asking? It removes the widget from the disabled_widgets which should make it clickable?

Reply:
Yes, it should just be enabled.

Reply:
I removed them, they are never set so it never needs removing, I think this is what you meant? It would be best to be very clear on what you are requesting in review comments also with a reasoning as to why, I was a bit confused by what you wanted, assuming this is what you wanted.